Tests/FluentAssertions.Shared.Specs/CollectionEquivalencySpecs.cs

```diff
@@ -17,6 +17,128 @@ namespace FluentAssertions.Specs
     [TestClass]
     public class CollectionEquivalencySpecs
     {
+        #region General
```

Comment:
There are six tests in this _region_. Three of them are "single object" overload tests while others are not.

Please check:
- Included new general tests for `IEnumerable` overload in order to see if both work as expected

I think those new tests (subject-null-throw, expectation-null-throw, suject-expectation-null-succeed) add value to the fixture escpecially with the new single object tests which doesn't have the same expectation for all given cases.

How should I arrange them? I can move new single object tests to a new region called "Single Object" and keep the rest (above) in "General" region.